packages/components/Notification/src/Notification.tsx

```diff
@@ -46,7 +47,8 @@ export const Notification = compose<NotificationType>({
     const Slots = useSlots(userProps, (layer) => notificationLookup(layer, userProps));
     const isBar = ['primaryOutlineBar', 'primaryBar', 'neutralBar'].includes(userProps.variant);
     const width = useWindowDimensions().width / 2;
-    const sizeClass = useHorizontalSizeClass();
+    const rootTag = useContext(RootTagContext);
+    const sizeClass = useHorizontalSizeClass(rootTag);
```

Comment:
Can the hook `useHorizontalSizeClass` just call `useContext(rootTagContext) for us so we don't need this extra line for all callers?

Reply:
Ooh I hadn't thought of that, will try